src/sider/skjema-registrering/skjema-sporsmalene.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
```

Comment:
Kan være greit å rename filen til f.eks registrering-sporsmal.tsx, eller noe annet, siden skjema-*, brukes nå for definering av skjemaer.